docs/src/logistic.md

```diff
@@ -2,13 +2,13 @@
 
 
 ```julia
-using Distributions
+using Distributions, TensorFlow, PyCall, MacroTools
```

Comment:
Why do you need PyCall and MacroTools?

Reply:
I just re-tested the old code and now with PyCall and MacroTools not called for the code works. I'm not able to explain why the situation was different the first time.